i18n/id.toml

```diff
@@ -0,0 +1,195 @@
+
+# === baseof ==
+[backToTop]
+other = "Kembali ke Atas"
+
+[viewComments]
+other = "Lihat Komentar"
+# === baseof ==
+
+# === Post ===
+[posts]
+other = "Pos"
+# === Post ===
+
+# === Taxonomy ===
+[allSome]
+other = "Semua {{ .Some }}"
+
+[tag]
+other = "Label"
+
+[tags]
+other = "Label"
+
+[category]
+other = "Kategori"
+
+[categories]
+other = "Kategori"
+# === Taxonomy ===
+
+# === Pagination ===
+[more]
+other = "Lainnya"
+# === Pagination ===
+
+# === partials/header.html ===
+[selectLanguage]
+other = "Pilih Bahasa"
+
+[switchTheme]
+other = "Ganti Tema"
+# === partials/header.html ===
+
+# === partials/footer.html ===
+[poweredBySome]
+other = "Ditenagai oleh {{ .Hugo }} | Tema - {{ .Theme }}"
+# === partials/footer.html ===
+
+# === partials/comment.html ===
+[valineLang]
+other = "en"
+
+[valinePlaceholder]
+other = "Komentarmu ..."
+
+[facebookLanguageCode]
+other = "id"
+# === partials/comment.html ===
+
+# === partials/assets.html ===
+[search]
+other = "Pencarian"
+
+[searchPlaceholder]
+other = "Cari judul atau konten ..."
+
+[clear]
+other = "Bersihkan"
+
+[cancel]
+other = "Batal"
+
+[noResultsFound]
+other = "Hasil tidak ditemukan"
+
+[lunrLanguageCode]
+other = "en"
+
+[copyToClipboard]
+other = "Salin ke papan klip"
+
+[cookieconsentMessage]
+other = "Situs ini menggunakan Cookies untuk meningkatkan pengalaman Anda."
+
+[cookieconsentDismiss]
+other = "Mengerti!"
+
+[cookieconsentLink]
+other = "Pelajari lebih lanjut"
+# === partials/assets.html ===
+
+# === partials/plugin/share.html ===
+[shareOn]
+other = "Bagikan ke"
+# === partials/plugin/share.html ===
+
+# === posts/single.html ===
+[contents]
+other = "Daftar Isi"
+
+[publishedOnDate]
+other = "diterbitkan pada {{ .Date }}"
+
+[includedInCategories]
+other = "di {{ .Categories }}"
+
+[wordCount]
+one = "Satu kata"
+other = "{{ .Count }} kata"
+
+[readingTime]
+one = "Satu menit"
+other = "{{ .Count }} menit"
+
+[views]
+other = "dilihat"
+
+[author]
+other = "Penulis"
+
+[updatedOnDate]
+other = "Diperbarui pada {{ .Date }}"
+
+[readMarkdown]
+other = "Lihat versi Markdown"
+
+[back]
+other = "Kembali"
+
+[home]
+other = "Beranda"
+
+[readMore]
+other = "Baca Selengkapnya"
+# === posts/single.html ===
+
+# === 404.html ===
+[pageNotFound]
+other = "Halaman tidak ditemukan"
+
+[pageNotFoundText]
+other = "Maaf, halaman yang Anda cari tidak ada."
+# === 404.html ===
+
+# === shortcodes/admonition.html ===
+[note]
+other = "Catatan"
+
+[abstract]
+other = "Abstrak"
+
+[info]
+other = "Info"
+
+[tip]
+other = "Kiat"
+
+[success]
+other = "Sukses"
+
+[question]
+other = "Pertanyaan"
+
+[warning]
+other = "Peringatan"
+
+[failure]
+other = "Kegagalan"
+
+[danger]
+other = "Bahaya"
+
+[bug]
+other = "Bug"
+
+[example]
+other = "Contoh"
+
+[quote]
+other = "Kutipan"
+# === shortcodes/admonition.html ===
+
+# === shortcodes/version.html ===
+[new]
+other = "BARU"
+
+[changed]
+other = "BERUBAH"
+
+[deleted]
+other = "DIHAPUS"
```

Comment:
_deleted_ is a past tense, so it should be `TERHAPUS`

Reply:
Or maybe `TELAH DIHAPUS`